runtime/opt/taupage/init.sh

```diff
@@ -75,9 +75,12 @@ else
     EC2_AVAIL_ZONE=$( curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone )
     EC2_REGION="`echo \"$EC2_AVAIL_ZONE\" | sed -e 's:\([0-9][0-9]*\)[a-z]*\$:\\1:'`"
 
-    echo "INFO: Notifying CloudFormation (region: $EC2_REGION, stack: $config_notify_cfn_stack, resource: $config_notify_cfn_resource, status: $result)..."
+    IAM_ROLE=$( curl -s http://169.254.169.254/latest/meta-data/iam/security-credentials )
```

Comment:
This needs `--fail` and exit status checks.

Reply:
It will, below the cfn-signal will short circuit and fallback to the current mode

Reply:
Yeah, but if the metadata API is unreachable I'd prefer a clear error message instead of having to figure out why `cfn-signal` failed.

Reply:
You mean, like we've been doing on line 75 ;-)

Reply:
Apart from the calls to the metadata endpoint, do you suggest any other improvements to the script?

Reply:
Right, the whole script is pretty bad. :(

Reply:
Please sign the commits so the DCO check would pass.